ingest-app/src/cmr/ingest/runner.clj

```diff
@@ -1,16 +1,14 @@
 (ns cmr.ingest.runner
   "Entry point for the application. Defines a main method that accepts arguments."
   (:require [cmr.ingest.system :as system]
-            [clojure.string :as string]
-            [cmr.common.log :as log :refer (debug info warn error)]
-            [cmr.common.api.web-server :as web]
-            [cmr.ingest.api.routes :as routes]
+            [cmr.common-app.app-events :as events]
             [cmr.common.config :as cfg])
   (:gen-class))
 
 (defn -main
   "Starts the App."
   [& args]
   (let [system (system/start (system/create-system))]
-    (info "Running...")
```

Comment:
Why remove this log event? (same question in multiple places).  I assume you are cleaning up, guess I'll just ask you to confirm that you are removing redundant messaging then.

Reply:
it is redundant with the alerts in common-lib/src/cmr/common/system.cl